recipes/kalpy/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "kalpy" %}
+{% set version = "0.0.6" %}
+
+package:
+  name: kalpy
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/k/kalpy-kaldi/kalpy-kaldi-{{ version }}.tar.gz
+  sha256: 70884c243c2eb44423e748af18c9c9a0730e614e88bc4fe7dccade945fe69111
+
+build:
+  number: 0
+  skip: True  # [py<38]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - python                                  # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}      # [build_platform != target_platform]
+    - cmake
+    - make                                    # [not win]
+    - ninja
+    - pybind11
+    - setuptools
+    - setuptools_scm
+  host:
+    - python
+    - pip
+    - kaldi
+    - openfst
+  run:
+    - python
+    - pynini
+    - kaldi
+    - numpy >=1.x
+    - librosa
+
+test:
+  requires:
+    - kaldi * cpu*
```

Please add a comment why we're adding this requirement. Also, prefer `=` over semantically relevant whitespace.
```suggestion
  requires:
    # the cuda variant of kaldi requires loading libcuda.so, which we don't have in regular CI
    - kaldi =*=cpu*
```